example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
```

Comment:
Let's rethink this. Could this type of cache be used for something else than identifying the application? Then "app cache" might be a bit misleading or not general enough.

Generally speaking: The cache caches tuples of DNs and attribute values after a search - right? Is the realm mapping already applied? Then it rather might be a "realm cache" than an "app cache". Or an "attribute cache"....